applications/CoSimulationApplication/tests/test_mok_fsi.py

```diff
@@ -42,6 +43,32 @@ def test_mok_fsi_aitken(self):
             self.__DumpUpdatedCFDSettings()
             self._runTest()
 
+    def test_mok_fsi_block_mvqn(self):
+        self.accelerator_type = "block_mvqn"
+
+        with KratosUnittest.WorkFolderScope(".", __file__):
+            self._createTest("fsi_mok", "cosim_mok_fsi_block")
+            self.__ManipulateCFDSettings()
+            self.__RemoveOutputFromCFD() # comment to get output
+            self.__AddTestingToCFD()
+            self.__DumpUpdatedCFDSettings()
+            self._runTest()
+
+
+    def test_mok_fsi_block_ibqnls(self):
+        self.accelerator_type = "block_ibqnls"
+        self.err_tol = "6e-5"
```

Comment:
When I test on multi threads, the error is a little larger than 1e-6, the results differ each time the the test is run. It's probably linked to the scipy operations (since it's larger than the error with `block_mvqn`) in the accelerator but I can't figure out where exactly. The results are the same when `OMP_NUM_THREADS=1`

Reply:
if you are sure which operation it is, then we could dynamically set the omp-threads to 1 for just this function, what do you think?

Reply:
I can't figure out which operation specifically, but my guess it's the combination of `sp.linalg.solve_triangular` and `sp.sparse.linalg.gmres`. But it's the "pseudo-inverse" operation itself and not just the functions, because I tested with a brute inversion of matrices and the error still varies with the same magnitude.
Setting omp-threads to 1 is fine too. But the error tolerance should still stay bigger than the other accelerators, because of the difference of results between different OS.

Reply:
Ok

FYI if you want to limit this accelerator to always use one thread, you could use sth like I did here:
https://github.com/KratosMultiphysics/Kratos/blob/c252d518b6f7bb441322e46daded6708a9263260/applications/CoSimulationApplication/python_scripts/solver_wrappers/kratos/kratos_base_wrapper.py#L16-L29